ion/src/device/shared/usb/dfu_interface.cpp

```diff
@@ -188,7 +189,7 @@ void DFUInterface::changeAddressPointerIfNeeded() {
 
 void DFUInterface::eraseCommand(uint8_t *transferBuffer, uint16_t transferBufferLength) {
   /* We determine whether the commands asks for a mass erase or which sector to
-   * erase. The erase must be done after the next getStatus request. */
+ * erase. The erase must be done after the next getStatus request. */

```

Comment:
?

Reply:
I think it have been sqashed when merging, but this fork isn't up-to-date, so it isn't corrected in this repo